src/Resources/views/CRUD/Association/edit_one_script.html.twig

```diff
@@ -39,13 +39,13 @@ This code manages the one-to-many association field popup
                 'elementId': id,
                 'objectId': sonata_admin.admin.root.id(sonata_admin.admin.root.subject),
                 'uniqid': sonata_admin.admin.root.uniqid,
-                'subclass': app.request.query.get('subclass'),
```

Comment:
this is not needed anymore?

Reply:
It's already inside the `app.request.query.all` value.